.github/workflows/gh-pages.yaml

```diff
@@ -124,14 +124,14 @@ jobs:
         uses: jacobtomlinson/gha-find-replace@v2
```

Comment:
Did you test in your mock-up with an already pre-existing stable version?

Reply:
That's good! But just to be clear, my question was: Have you tried already having a stable/ version in your mockup, then adding a new release?  We want to verify this overwrites the existing stable/ folder with the new release's files.

Reply:
Yeap, that's right. In pseudocode, here's what happens:

**In cleanlab/cleanlab repo:**

1. A new release tag, say v2.1.0, triggers the gh-pages.yaml workflow
2. "v2.1.0" gets written to a file called "latest_release.txt"
3. this file gets pushed to the cleanlab/cleanlab-docs repo.

**In cleanlab/cleanlab-docs repo:**

4. this push triggers the stable-dir.yaml
5. everything in the "stable" dir gets deleted
6. content in the "v2.1.0" dir gets copied over to the "stable" dir
7. push back to cleanlab/cleanlab-docs